modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonAbstractConnexionServerCodegen.java

```diff
@@ -304,7 +315,20 @@ public String escapeReservedWord(String name) {
      */
     @Override
     public String apiFileFolder() {
-        return outputFolder + File.separator + apiPackage().replace('.', File.separatorChar);
+        String pkgPath = apiPackage().replace('.', File.separatorChar);
+        if (this.srcLayout) {
+            pkgPath = "src" + File.separator + pkgPath;
+        }
```

Comment:
Why not edit outputFolder directly when we assign it first?
Then we don't need to put these fi blocks in twice.
Up in processOpts:
```
        if (this.srcLayout) {
            outputFolder = "src" + File.separator + "src";
        }
```

Reply:
Because there are still supporting files (like Readme, .gitignore, etc) that still need to be in the root dir. So we need to selectively move only the source files under the src/ dir.

Reply:
Ah, in that case why not use your new variable
pythonSrcRoot? Then it is only defined once and it will include the file separator when you pass in the value "src/"

Reply:
done